cmd-export-attachments.c

```diff
@@ -31,13 +31,6 @@
 
 #include "sigbak.h"
 
-#define FLAG_EXPORT_ALL		0x1
-#define FLAG_FILENAME_ID	0x2
```

Comment:
FLAG_FILENAME_ID is also needed in cmd-export-messages.c now. I moved all of those FLAG_* to keep them together.